integration/conftest.py

```diff
@@ -274,6 +322,7 @@ def bob(reactor, temp_dir, introducer_furl, flog_gatherer, request):
     return pytest_twisted.blockon(
         MagicFolderEnabledNode.create(
             reactor,
+            tahoe_venv,
```

Comment:
It looks like this fixture isn't requested here or below. I *think* this is actually passing the fixture function, rather than the resolved fixture.